scripts/cre_sync/docs/requirements.md

```diff
@@ -0,0 +1,428 @@
+CRE Utilities Implementation
+===
+
+Introduction
+---
+
+In order to implement the CRE v0.91 design defined [here|] a web based application needs to be created.
+This application needs to be easily deployable and also define a feature-complete library which would allow for others to adopt the CRE structure.
+
+The CRE Design linked to above, lists the following components:
+
+* Sheet Parser
+* Source Parser
+* CRE WebApp
+* CRE query engine
+
+It also lists the following requirements:
+
+[Requirement] Source Parser
+---
+
+A CRE source document, needs to include the following information as a minimum:
+
+``` Yaml
+
+# Mapping file WSTG
+
+Segment short name (optional): CRYP-04
+Segment long name (optional): Testing for weak encryption
+URL: https://owasp.org/www-project-web-security-testing-guide/v41/4-Web_Application_Security_Testing/09-Testing_for_Weak_Cryptography/04-Testing_for_Weak_Encryption.html
+CRE-ID: 273-300. (In case a segment links to multiple CRE's, multiple entries can be specified, or the CRE-ID's can be enumerated in the CRE-ID field. For example: CRE-ID: 273-300, 251-823)
```

Comment:
yes this is a CRE array, that\'s pretty much preserved in the File Format section of the implementation, however instead of only providing the CRE-IDs the user would have to provide a full document